src/mkdocs_include_markdown_plugin/event.py

```diff
@@ -212,7 +231,10 @@ def found_include_tag(match: re.Match[str]) -> str:
             )
             return ''
         elif files_watcher is not None:
-            files_watcher.included_files.extend(file_paths_to_include)
+            if not is_url(file_path_glob):
+                files_watcher.included_files.extend(file_paths_to_include)
+            else:
+                print('not adding a watcher')
```

Comment:
Could you improve these `print` statements as log records?

Reply:
fixed, please review